<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YoshiSO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Descargas/Yoshi.png</normaloff>../../Descargas/Yoshi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="IniProcGP">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="title">
           <string>Procesos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="noProcLE">
             <property name="placeholderText">
              <string>No. Procesos</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="noProcePB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageResPro">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="2" column="1">
         <widget class="QGroupBox" name="executingprocGP">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="restTimeLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Tiempo restante: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="onIdLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="passTimeLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Tiempo transcurrido: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="operationLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Operaci√≥n:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="onOperatorLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="estTimeLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Tiempo estimado:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="onEstimatedTimeLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="idLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="onPassTimeLB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="onRestTimeB">
             <property name="font">
              <font>
               <family>Rockwell</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="executingProcLB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Procesos ejecutandose</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="unfinishedBatchLB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Procesos  Pendientes: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="finishedProcLB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Procesos finalizados</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="globalCountLB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Contador global:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QScrollArea" name="finishedProcSA">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>396</width>
             <height>388</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QGridLayout" name="finishedProcGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QGridLayout" name="pendingProcGL"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="startPB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pendingProcLB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Listos</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="blocked_LB">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cola de bloqueados</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QScrollArea" name="blocked_SA">
          <property name="maximumSize">
           <size>
            <width>1500</width>
            <height>120</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1063</width>
             <height>118</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="1" column="0">
             <layout class="QGridLayout" name="blocked_GL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btnFinish">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Terminar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="resultPage">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QScrollArea" name="resultSA">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>66</width>
             <height>20</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="resultGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
